tests/skybrightness_pre/test_ZernikeSky.py

```diff
@@ -47,6 +47,8 @@ def test_bulk_zernike_fit(self):
         self.assertGreater(reread_zernike.shape[1], 20)
         self.assertEqual(tuple(reread_zernike.index.names), ("band", "mjd"))
 
+        test_out_dir.cleanup()
```

Comment:
This shouldn't be necessary. TemporaryDirectory objects automatically clean themselves up when the go out of scope. An explicit cleanup should only be necessary if you want to delete the directory before the instance of TemporaryDirectory goes out of scope (or is explicitly deleted).

Reply:
You’re right that it cleans up itself, but it also gives a warning while doing so. This removed the warning message.